File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Dickins/DickinsFlatBray.m
clear all, close all

addpath /home/orodrig/FORdoc/at/Matlab/Plot
addpath /home/orodrig/FORdoc/at/Matlab/ReadWrite

disp('Bellhop - Dickins Seamount:') 
disp('Ray trace run')
disp('Geometric hat beams in Cartesian coordinates') 

system('bellhop.exe DickinsFlatBray');

plotray('DickinsFlatBray')

disp('done.')

File: acoustics_toolbox_r2020/Python/Bellhop/Tests/DW/Dickins/DickinsFlatBray.py
#=======================================================================
# 
# Bellhop: Dickins Seamount
# Mexilhoeira Grande, Dom Jul 30 11:42:34 WEST 2017
# Written by Tordar 
# 
#=======================================================================

from os import system
from numpy import *
from scipy.io import *
from pylab import *
sys.path.append ("/home/orodrig/FORdoc/at/Python/")
from plotray import *

print("Bellhop - Dickins Seamount:") 
print("Ray trace run")
print("Geometric hat beams in Cartesian coordinates") 

system("bellhop.exe DickinsFlatBray")

figure(1)
plotray('DickinsFlatBray.ray')
xlabel('Range (m)')
ylabel('Depth (m)')
title("Bellhop - Dickins Seamount")

show()

print("done.")

